### PR TITLE
feat(pluginutils): support bigint and symbol in dataToEsm

### PR DESCRIPTION
**Pull request by LongTengDao**
## Rollup Plugin Name: `pluginutils`

This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers:

### Description

|data|oldEsm|newEsm|
|---|---|---|
|`bigint`|`undefined`|`bigint`|
|`Symbol.for()`|`undefined`|`Symbol.for()`|

### CODE REVIEW

File: packages/pluginutils/test/dataToEsm.ts [hunk 1]
```diff
@@ -2,6 +2,13 @@ import test from 'ava';
 
 import { dataToEsm } from '../';
 
+test('support bigint and symbol', (t) => {
```
Comment (shellscape) [inline packages/pluginutils/test/dataToEsm.ts:5]: please split these into separate tests. (that may seem pedantic, but this PR essentially adds two features in support for two different entities)

File: packages/pluginutils/test/dataToEsm.ts [hunk 2]
```diff
@@ -2,6 +2,13 @@ import test from 'ava';
...
+  t.is(
+    dataToEsm({ bigint: 0, symbol: Symbol.for('') }),
```
Comment (shellscape) [inline packages/pluginutils/test/dataToEsm.ts:7]: I'd like to see a few different values (within the same `t` test func) tried here for sanity check

File: packages/pluginutils/src/dataToEsm.ts [hunk 3]
```diff
@@ -4,8 +4,8 @@ import makeLegalIdentifier from './makeLegalIdentifier';
 
 export type Indent = string | null | undefined;
 
-function stringify(obj: unknown): string {
-  return (JSON.stringify(obj) || 'undefined').replace(
+function stringify(val: boolean | number | string): string {
```
Comment (shellscape) [inline packages/pluginutils/src/dataToEsm.ts:7]: why is this change necessary?

**Review by shellscape — CHANGES_REQUESTED** [review]

**Comment by shellscape** [comment]
Looks like there are some failures caused by these changes in CI. Please have a look

File: packages/pluginutils/src/dataToEsm.ts (hunk 3 above)
Comment (LongTengDao) [reply packages/pluginutils/src/dataToEsm.ts:7]: because rest types (`undefined` `object` `symbol` `bigint`) already and should be handled in `serialize`, I think this will make the code more strong.

if it's not necessary, this change could be reverted.

Comment (LongTengDao) [reply packages/pluginutils/src/dataToEsm.ts]: I reverted this change for CI undue error.

**Review by shellscape — APPROVED** [review]

**Comment by shellscape** [comment]
Closing as abandoned. The error is visible here: https://github.com/rollup/plugins/runs/5281573593?check_suite_focus=true#step:7:241. If you get around to cleaning that up, please update your fork and give us a ping.

**Comment by LongTengDao** [comment]
@shellscape Oh, sorry, I missed. The typo error has been corrected.

**Comment by shellscape** [comment]
Thanks for updating it!

**Comment by shellscape** [comment]
Looks like there is a failing test to take a look at.

**Comment by LongTengDao** [comment]
@shellscape Thanks for your careful guidance
